test/built-ins/Temporal/PlainYearMonth/prototype/subtract/calendar-arguments-extra-options.js

```diff
@@ -31,7 +31,8 @@ class CustomCalendar extends Temporal.Calendar {
   }
   dateAdd(date, duration, options) {
     const result = super.dateAdd(date, duration, options);
-    options.overflow = 'meatloaf';
+    if (options)
+      options.overflow = 'meatloaf';
```

Comment:
I didn't realize that this was a side effect of the normative change. Nonetheless, I think it's probably still the correct change to make. It seems to me that the `dateAdd` calls for finding the last day of the month need to have the default options.

You could probably sidestep this by changing the test to subtract -5 months below instead of 5 months.

Reply:
How about adding a `dateAddCalled` counter, and putting a clearer condition? like:
```javascript
    dateAddCalls += 1
    if (dateAddCalls == 3)
      options.overflow = 'meatloaf';
```

Reply:
> How about adding a `dateAddCalled` counter, and putting a clearer condition? like:
> 
> ```js
>     dateAddCalls++;
>     if (dateAddCalls == 3)
>       options.overflow = 'meatloaf';
> ```

I've just pushed a change doing this